packages/ui-shared/src/components/TextWithIcon/TextWithIconItem.tsx

```diff
@@ -1,9 +1,14 @@
 import * as React from 'react';
-import { cnCreate, filterDataAttrs } from '@megafon/ui-helpers';
+import { cnCreate, convert, filterDataAttrs, textConvertConfig } from '@megafon/ui-helpers';
 import * as PropTypes from 'prop-types';
 import './TextWithIconItem.less';
 
 export interface ITextWithIconItem {
+    /** Дополнительные классы для корневого и внутренних элементов */
```

Comment:
Ты не задаешь для корневого элемента класс. Либо задай его, либо так:
```suggestion
    /** Дополнительные классы для внутренних элементов */
```